networking/install.bash

```diff
@@ -1,9 +1,13 @@
 #! /usr/bin/env bash
 
-# Install config file (mdns4_minimal is normally missing and should be present)
+. /etc/os-release
+if [ $UBUNTU_CODENAME = "xenial" ]
+then
+    # Install config file (mdns4_minimal is normally missing in xenial and should be present)
 
-# Replace nsswitch config file
-tue-install-cp nsswitch.conf /etc/nsswitch.conf
+    # Replace nsswitch config file
+    tue-install-cp nsswitch.conf /etc/nsswitch.conf
```

Comment:
Is this not needed in 18.04+?

Reply:
Nope. It's always been a bug, just too bad it landed in an LTS version of ubuntu.
Fixing it would be non-backwards compatible so they didn't in 16.04